test/acceptation/app.js

```diff
@@ -6,17 +6,18 @@ const path = require('path');
 
 describe('Accept Application Test', function() {
     it('Should work from A to Z', function() {
-        this.timeout(4000);
-        let configPath = path.join(__dirname, '..', 'integration', 'artifacts', 'application', '.pferc');
+        this.timeout(5000);
```

Comment:
At this point make this test 50000 hahahaha

Reply:
Just to be sure travis would not explode.

Reply:
Yeah that's my point hahaha Travis will not explode for a while if we make it 50000